screens/Login.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { TouchableOpacity, TextInput, Text, View, StyleSheet } from 'react-native'
+
+export const Login = ({ navigation }) => {
+
+  const [userName, setUserName] = useState('')
+  const [password, setPassword] = useState('')
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.welcomeText}>Welcome to your App</Text>
+      <View style={styles.inputView}>
+        <TextInput 
+          // required
+          style={styles.inputText}
+          placeholder="Username"
+          placeholderTextColor="#fff"
+          onChangeText={value => setUserName(value)}
+          defaultValue={userName}
+        />
+      </View>
+      <View style={styles.inputView}>
+        <TextInput 
+          // required
+          style={styles.inputText}
+          placeholder="Password"
+          placeholderTextColor="#fff"
+          onChangeText={value => setPassword(value)}
+          defaultValue={password}
+        />
+      </View>
+      <TouchableOpacity>
+        <Text style={styles.forgot}>Forgot Password?</Text>
+      </TouchableOpacity>
+      <TouchableOpacity 
+        style={styles.loginButton}
+        title="Login" 
+        onPress={() => navigation.navigate("Profile", {userName : userName})} 
+      >
+        <Text style={styles.loginText}>LOGIN</Text>
+      </TouchableOpacity>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
```

Comment:
You should use Styled-Component in this project. For more info about it you can visit: https://styled-components.com/